lang/CMakeLists.txt

```diff
@@ -293,6 +293,9 @@ endif()
 if (WIN32)
 	target_link_libraries(libsclang wsock32 ws2_32)
     target_link_libraries(libsclang portmidi)
+    target_link_libraries(libsclang kernel32 user32 gdi32 winspool shell32 
+    	ole32 oleaut32 uuid comdlg32 advapi32 setupapi hidapi hidapi_parser 
+    	libscsynth tlsf boost_thread boost_system winmm yaml)
```

Comment:
i don't have other systems to test on. those libraries were not being linked on windows.

i was minimizing impact of my changes to the other platforms.